### PR TITLE
Restructured the repo and updated the Cheatsheet

### PR DESCRIPTION
**Pull request by sukantsondhi**
Added a homepage for the repo to make it more interactive and also updated the GHCP cheatsheet with the latest commands and extensions.

### CODE REVIEW

**Review by cmendible — CHANGES_REQUESTED** [review]
Hi there, let's keep the repo tree simple. Please remove the `GitHub Copilot Playground` folder and rebase your branch with the latest changes.

**Comment by sukantsondhi** [comment]
No worries. will do!! Would you like me to add some quick links for GHCP in the original README or not?

**Comment by cmendible** [comment]
> No worries. will do!! Would you like me to add some quick links for GHCP in the original README or not?

Sure add the quick links.

Thanks!

File: challenges/cryptoanalisis/cryto.zip
Comment (cmendible) [inline challenges/cryptoanalisis/cryto.zip:1]: This file should not be renamed.

**Review by cmendible — CHANGES_REQUESTED** [review]

File: challenges/cryptoanalisis/cryto.zip
Comment (sukantsondhi) [reply challenges/cryptoanalisis/cryto.zip:1]: That file in untouched and hasn't been renamed

**Review by cmendible — APPROVED** [review]
LGTM

**Comment by cmendible** [comment]
@sukantsondhi please rebase your branch and resolve any conflicts before we can merge this one

**Pull request merged**